### PR TITLE
Add players to championships

### PR DESCRIPTION
**Pull request by dominikw**
Players::BatchValidator needs some improvements - there is a need to check for uniqueness.

### CODE REVIEW

File: app/services/players/batch_validator.rb [hunk 1]
```diff
@@ -0,0 +1,28 @@
+module Players
+  class BatchValidator
+    def initialize(params)
+      @params = params
+    end
+
+    def failure?
+      no_players? || players_invalid?
```
Comment (mateuszgorniak) [inline app/services/players/batch_validator.rb]: I'd do something like this:
```
@messages = no_players
@messages ||= players_invalid
!@mesages.nil?
```

or even using separate Validator for Batches - the best option IMHO

File: app/services/players/batch_creator.rb [hunk 2]
```diff
@@ -0,0 +1,15 @@
+module Players
+  class BatchCreator
+    def initialize(params)
+      @params = params
+    end
+
+    def call
+      params.each { |player_params| Player.create(player_params) }
```
Comment (mateuszgorniak) [inline app/services/players/batch_creator.rb]: is there a possibility to use batch create in Sequel?

Comment (dominikw) [inline app/services/players/batch_creator.rb]: There is - `multi_insert`, thanks!

**Comment by mateuszgorniak** [comment]
Is it still `in progress`, @dominikw ?

**Pull request merged**